dm/proto/dmmaster.proto

```diff
@@ -3,14 +3,29 @@ syntax = "proto3";
 package pb;
 
 import "dmworker.proto"; // refine if needed
-
+import "google/api/annotations.proto";
 
 service Master {
-    rpc StartTask (StartTaskRequest) returns (StartTaskResponse) {}
-    rpc OperateTask (OperateTaskRequest) returns (OperateTaskResponse) {}
+    rpc StartTask (StartTaskRequest) returns (StartTaskResponse) {
+        option (google.api.http) = { 
+            post: "/apis/alpha/tasks"
```

Comment:
is something like `v1` needed in the path? and can we have multi versions exist at the same time?

Reply:
seems can have multi versions (path) with <https://github.com/googleapis/googleapis/blob/db02bc9c107268372f256dd72ebeecfa20c06007/google/api/http.proto#L183>.

Reply:
it is used to bind more than one urls for one request?

Reply:
`alpha` is the version, after the interface is positive we can change it to v1

Reply:
> it is used to bind more than one urls for one request?

seems, yes.

Reply:
> `alpha` is the version, after the interface is positive we can change it to v1

are there `v1alpha`, `v2alpha`, etc.?

Reply:
change to `v1alpha1` at https://github.com/pingcap/dm/pull/224/commits/a42a028cefd50dfe5feb9d8609a41a8f94ea10cf